src/parcel_snoopi.jl

```diff
@@ -227,6 +227,14 @@ function parcel(tinf::AbstractVector{Tuple{Float64,Core.MethodInstance}}; subst=
         # Use special care with keyword functions, anonymous functions
         p = tt.parameters[1]   # the portion of the signature related to the function itself
         paramrepr = map(T->reprcontext(topmod, T), Iterators.drop(tt.parameters, 1))  # all the rest of the args
+
+        # blacklist remover
+        for (iLine, line) in enumerate(pc[topmodname])
+          if any(occursin.(blacklist, line))
+            deleteat!(pc[topmodname], iLine)
```

Comment:
This will affect later iterations in the enumeration. Demo:
```julia
julia> list = [1, 2, 3, 3, 4, 5]
6-element Array{Int64,1}:
 1
 2
 3
 3
 4
 5

julia> function remove3!(list)
           for (i, v) in enumerate(list)
               v == 3 && deleteat!(list, i)
           end
           return list
       end
remove3! (generic function with 1 method)

julia> remove3!(list)
5-element Array{Int64,1}:
 1
 2
 3
 4
 5
```
Only the first 3 is removed because the index advanced past the one that followed it.

Instead, accumulate all the indexes you want to delete to temporary storage and delete them at the conclusion of the loop.

Reply:
Ahhh. That's right. :smile: I didn't notice that. I will fix it